<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStudent</class>
 <widget class="QWidget" name="addStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">BACKGROUND-COLOR:#06113C;
color: rgb(255, 255, 255);
font: 13pt &quot;Optima&quot;;
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">COLOR:white;

font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>First name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Fname">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);

	

	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;
</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>Last name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Sname">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);


	

	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;
</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;
</string>
       </property>
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_age">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);
	

	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;
</string>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);
	

	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;
</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;
</string>
       </property>
       <property name="text">
        <string>phone number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_phone">
       <property name="styleSheet">
        <string notr="true">
BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);
	

	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;
</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;
</string>
       </property>
       <property name="text">
        <string>Grade</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_grade">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
BACKGROUND:rgb(255, 255, 255);
ont: 13pt &quot;Optima&quot;;

	



	</string>
       </property>
       <item>
        <property name="text">
         <string>Prep</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>First</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Second</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Third</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Forth</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>37</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_addStudent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 13pt &quot;Optima&quot;;
background-color: #FF8C32;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 17px;
	border-color: #f0742f;
	padding: 8px;

</string>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>37</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
